pkgs/sagemath-doc-html/meson.build

```
@@ -0,0 +1,30 @@
+project('sagemath-doc-html',
+        version: files('VERSION.txt')
+)
+
+# This is a meson-python project, so this is subject to the wheel mappings
+# defined in https://github.com/mesonbuild/meson-python/blob/main/mesonpy/__init__.py
+docdir = get_option('datadir') / 'share' / 'doc' / 'sage'
+
+doc_src = custom_target('Writable documentation source tree',
+  output: 'doc-src',  # phony
+  command: [meson.current_source_dir() / 'copy_doc_src.py'],
+  env: {'SAGE_DOC': meson.current_build_dir(),
+        'SAGE_DOC_SRC': meson.current_source_dir() / 'doc'}
+)
+
+html = custom_target('HTML documentation',
+  output: 'index.html',
+  depends: [doc_src],
+  command: [meson.current_source_dir() / 'make.sh',
```

```suggestion
  console: true,
```

Results in a better output (e.g no output buffering).

---

Thanks for finding this. I've added it in 8f09f52062d